source/lib-intro.tex

```diff
@@ -1138,12 +1138,12 @@
 standard header \tcode{<iso646.h>} or \tcode{<ciso646>} has no effect.}
 
 \pnum
-\ref{depr.c.headers}, C standard library headers, describes the effects of using
-the \tcode{\placeholder{name}.h} (C header) form in a \Cpp program.\footnote{ The
+Annex~\ref{depr.c.headers}, C standard library headers, describes the effects of using
```

Comment:
I'm also not sure if we can actually call it "Annex" if it refers to a strict subsection. Similarly, we only ever say "Clause" for the top-level chapters.